tests/test_categorical.py

```diff
@@ -86,6 +86,58 @@ def test(self):
 ]
 
 
+class ReplaceCategoricalColumnsTest(unittest.TestCase):
```

Comment:
It is better to make sure that non-categorical columns are not updated in `ReplaceCategoricalColumns`.

Reply:
Testing repeat substitutions give the same result for the same values.

Reply:
fc02ca8 @wafrelka